flink-runtime/src/main/java/org/apache/flink/runtime/state/ttl/AbstractTtlState.java

```diff
@@ -38,10 +37,12 @@
 	extends AbstractTtlDecorator<S>
 	implements InternalKvState<K, N, SV> {
 	private final TypeSerializer<SV> valueSerializer;
+	final Runnable accessCallback;
```

Comment:
I suggest a comment because it is not obvious for a reader of this class what this is good for.